draft-ietf-quic-http.md

```diff
@@ -1132,6 +1133,10 @@ an error code of H3_STREAM_CREATION_ERROR or a reserved error code
 ({{http-error-codes}}), but MUST NOT consider such streams to be a connection
 error of any kind.
 
+As certain stream types can affect connection state, a recipient SHOULD NOT
+discard data from incoming unidirectional streams prior to reading the type
+byte.
```

Comment:
Type can be longer than 1 byte (it's a varint)

Reply:
Same question about 'discard data' - is the client expected to read this stream until FIN/RST or can it abort?

Reply:
The client is expected to read until it knows the type of the stream.  At that point, it's free to abort or continue reading based on that information.